plugins/commands/winrm_config/command.rb

```diff
@@ -0,0 +1,124 @@
+require 'optparse'
+require "vagrant/util/safe_puts"
+
+module VagrantPlugins
+  module CommandWinRMConfig
+    class Command < Vagrant.plugin("2", :command)
+      include Vagrant::Util::SafePuts
+
+      def self.synopsis
+        "outputs WinRM configuration to connect to the machine"
+      end
+
+      def convert_win_paths(paths)
+        paths.map! { |path| Vagrant::Util::Platform.format_windows_path(path, :disable_unc) }
+      end
+
+      def execute
+        options = {}
+
+        opts = OptionParser.new do |o|
+          o.banner = "Usage: vagrant winrm-config [options] [name|id]"
+          o.separator ""
+          o.separator "Options:"
+          o.separator ""
+
+          o.on("--host NAME", "Name the host for the config") do |h|
+            options[:host] = h
+          end
+        end
+
+        argv = parse_options(opts)
+        return if !argv
+
+        with_target_vms(argv) do |machine|
+          winrm_info = CommunicatorWinRM::Helper.winrm_info(machine)
```

Comment:
This ends up as a stacktrace if you run the command on a guest that doesn't have this class loaded:

```
brian@localghost:vagrant-sandbox % be vagrant winrm-config bork                                          ±[●][master]
bundler: failed to load command: vagrant (/home/brian/.rvm/gems/ruby-2.5.1@vagrant-sandbox/bin/vagrant)
NameError: uninitialized constant VagrantPlugins::CommunicatorWinRM::Helper
  /home/brian/code/vagrant/plugins/commands/winrm_config/command.rb:35:in `block in execute'
  /home/brian/code/vagrant/lib/vagrant/plugin/v2/command.rb:238:in `block in with_target_vms'
  /home/brian/code/vagrant/lib/vagrant/plugin/v2/command.rb:232:in `each'
  /home/brian/code/vagrant/lib/vagrant/plugin/v2/command.rb:232:in `with_target_vms'
  /home/brian/code/vagrant/plugins/commands/winrm_config/command.rb:34:in `execute'
  /home/brian/code/vagrant/lib/vagrant/cli.rb:54:in `execute'
  /home/brian/code/vagrant/lib/vagrant/environment.rb:291:in `cli'
  /home/brian/code/vagrant/bin/vagrant:164:in `<top (required)>'
  /home/brian/.rvm/gems/ruby-2.5.1@vagrant-sandbox/bin/vagrant:23:in `load'
  /home/brian/.rvm/gems/ruby-2.5.1@vagrant-sandbox/bin/vagrant:23:in `<top (required)>'
```